Technetium.Web/Extensions/WebApplicationExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Technetium.Data;
+
+namespace Technetium.Web.Extensions;
+
+public static class WebApplicationExtensions
+{
+    public static async Task ApplyMigrationsAsync(this WebApplication application)
+    {
+        using var scope = application.Services.CreateScope();
+        var logger = scope.ServiceProvider.GetRequiredService<ILogger<Program>>();
+
+        if (logger.IsEnabled(LogLevel.Information))
+        {
+            logger.LogInformation("Migrating database");
+        }
```

Comment:
_(Minor)_ These checks for `IsEnabled` are not required.

The method in question already has the same check inside.

The check _may be_ required if the calculation of the log message costs us much. In case it's a constant, then the check is redundant.